src/setup-ros-osx.ts

```diff
@@ -27,7 +27,7 @@ export async function runOsX() {
 
 	// While rosdep and vcs are available as a Debian package on Ubuntu, they need
 	// to be installed through pip on OS X.
-	await pip.runPython3PipInstall(["rosdep", "vcstool"]);
+	await pip.runPython3PipInstall(["catkin-pkg", "rosdep", "vcstool"]);
```

Comment:
Looks like this install was missing, but I'm confused as to how this was not failing earlier?